### ISSUE TITLE
componentWillReceiveProps has been renamed, and is not recommended for use

### ISSUE THREAD

**Issue by breahna-c** [yuritoledo/react-native-credit-card-input#30]
Getting this warning in console:
```
Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: CCInput, LiteCreditCardInput
```

I'm using 
`"react-native-credit-card-input":  "0.4.8",`
`"react-native": "0.61.1",`

I could create a PR to suppress the warning, but I'm not sure if it's a good approach. 
I would do, as it recommends, move code to `componentDidUpdate`, won't it break the component?

What do you think?

**Comment by yuritoledo** [comment]
Hello,

I think it will not break, if you change to `componentDidUpdate`, and I agree, it's a better approach.

Could you make a PR?

**Comment by yuritoledo** [comment]
fixed in #38
